### ISSUE TITLE
ninja hitting the open-file limit.

### ISSUE THREAD

**Issue by tanin47** [google/clusterfuzz-tools#261]
```
2017-05-10 19:37:56,428 [DEBUG]: [612/31388] CC obj/native_client/src/trusted/validator_ragel/dfa_validate/validator_x86_64.o
2017-05-10 19:37:56,428 [DEBUG]: ninja: fatal: pipe: Too many open files
2017-05-10 19:37:56,428 [DEBUG]: ---------------------------------------
```

Adding `os.system('ulimit -a')` shows that the open-file limit is only 1024, even though we already increase the open-file limit through ansible.

**Comment by tanin47** [comment]
Adding `ulimit -n 1000000` before running `python-daemon` seems to work.

**Comment by tanin47** [comment]
Fixed with #262
